### ISSUE TITLE
Annotations of Training set

### ISSUE THREAD

**Issue by YiwuZhong** [TencentARC/Video-Holmes#4]
Thanks for sharing this solid work! 

I noticed that there is a folder of annotations. However, it seems to only cover the videos in test set. Could you please also share the annotations for training set videos? Thanks!

Yiwu

**Comment by donahowe** [comment]
Hi, We are currently organizing the relevant annotations and will open source them as soon as possible.

**Comment by YiwuZhong** [comment]
@donahowe Is there any estimated timeline for the annotation release?
